CHANGELOG.md

```diff
@@ -101,6 +101,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   now shorter within the default log revset. You can override the default by
   setting the `revsets.short-prefixes` config to a different revset.
 
+* The `@git` branches that track local branches of the local git repo are now
+  listed by `jj branch list`. Such branches exist in colocated repos or if you
+  use `jj git export`.
```

Comment:
I think "The `@git` branches" makes it sound like an existing concept. What do you think about instead writing it something like the following?

> The last seen state of branches in the underlying git repo is now presented by `jj branch list` as a remote called `git` (e.g. `main@git`). Such branches exist ...

Reply:
That sounds better, thanks!